packages/web-core/package.json

```diff
@@ -34,6 +34,7 @@
     "url": "https://github.com/corbado/javascript/issues"
   },
   "dependencies": {
+    "@fingerprintjs/fingerprintjs": "^4.4.3",
```

Comment:
![image](https://github.com/user-attachments/assets/08becc3d-2267-46bd-a9d3-b3fe00645b04)
https://github.com/fingerprintjs/fingerprintjs

To my understanding we can only use v3 because starting from v4 they made a change to the license which is no longer permissive.